mtags/src/main/scala-3/scala/meta/internal/pc/completions/OverrideCompletions.scala

```diff
@@ -171,8 +171,13 @@ object OverrideCompletions:
     val pos = driver.sourcePosition(params)
 
     val newctx = driver.currentCtx.fresh.setCompilationUnit(unit)
+    val tpdTree = newctx.compilationUnit.tpdTree
     val path =
-      Interactive.pathTo(newctx.compilationUnit.tpdTree, pos.span)(using newctx)
+      Interactive.pathTo(tpdTree, pos.span)(using newctx) match
```

Comment:
Previously in 
```scala 
trait NodeDb[F[_]]:
  type N
  ...

class InMemoryNodeDb[F[_]] extends NodeDb[F]
```
using `implementAll` `type N` was not seen in scope, so label would be `NodeDb.this.N`
Now, we see path as from the inside of `InMemoryNodeDb` class, and `type N` exists there.